x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -467,6 +479,23 @@ function FieldItemPopoverContents(props: State & FieldItemProps) {
         </EuiText>
       </>
     );
+  } else if (field.type === 'geo_point' || field.type === 'geo_shape') {
+    return (
+      <>
+        <EuiPopoverTitle>{panelHeader}</EuiPopoverTitle>
+
+        <EuiText size="s">{getVisualizeGeoFieldMessage(field.type)}</EuiText>
+
+        <EuiSpacer size="m" />
+        <EuiTextAlign textAlign="center">
+          <VisualizeGeoFieldButton
+            uiActions={uiActions}
+            indexPatternId={indexPattern.id}
+            fieldName={field.name}
+          />
+        </EuiTextAlign>
```

Comment:
Can maintain left alignment to be consistent with above text.

```suggestion
          <VisualizeGeoFieldButton
            uiActions={uiActions}
            indexPatternId={indexPattern.id}
            fieldName={field.name}
          />
```